go.mod

```diff
@@ -12,17 +12,18 @@ require (
 	github.com/alecthomas/chroma v0.8.1
 	github.com/go-chi/chi v3.3.2+incompatible
 	github.com/go-chi/render v1.0.1
-	github.com/google/uuid v1.1.2
+	github.com/google/uuid v1.3.0
 	github.com/gorilla/sessions v1.2.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/mitchellh/mapstructure v1.1.2
-	github.com/onflow/cadence v0.18.0
-	github.com/onflow/flow-go v0.18.2-canary
-	github.com/onflow/flow-go-sdk v0.20.0-alpha.1
+	github.com/mitchellh/mapstructure v1.4.1
+	github.com/onflow/atree v0.1.1
```

Comment:
how come this is added? is that a cadence dep?

Reply:
Yeah, the new version of Cadence depends on `atree` for the new storage layer